Implement font-variant-emoji #22118 (#22794)
mdn/content: files/en-us/web/css/font-variant/index.md

### Values

- `normal`
  - : Specifies a normal font face; each of the longhand properties has an initial value of normal. Longhand properties of `font-variant` are: [`font-variant-caps`](/en-US/docs/Web/CSS/font-variant-caps), [`font-variant-numeric`](/en-US/docs/Web/CSS/font-variant-numeric), [`font-variant-alternates`](/en-US/docs/Web/CSS/font-variant-alternates), [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures), [`font-variant-east-asian`](/en-US/docs/Web/CSS/font-variant-east-asian), [`font-variant-position`](/en-US/docs/Web/CSS/font-variant-position) and [`font-variant-emoji`](/en-US/docs/Web/CSS/font-variant-emoji).
- `none`

>   - : Sets the value of the [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures) to `none` and the values of the other longhand property as `normal`, their initial value.

Reviewer: ```suggestion
  - : Sets the value of the [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures) as `none` and the values of the other longhand properties as `normal`, their initial value.
```

- `<common-lig-values>`, `<discretionary-lig-values>`, `<historical-lig-values>`, `<contextual-alt-values>`

>   - : Specifies the keywords related to the [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures) longhand property. The possible values are: `common-ligatures`, `no-common-ligatures`, `discretionary-ligatures`, `no-discretionary-ligatures`, `historical-ligatures`, `no-historical-ligatures`, `contextual`, and `no-contextual`.

Reviewer: ```suggestion
  - : Specifies the keywords related to the [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures) longhand property. The possible values are `common-ligatures`, `no-common-ligatures`, `discretionary-ligatures`, `no-discretionary-ligatures`, `historical-ligatures`, `no-historical-ligatures`, `contextual`, and `no-contextual`.
```
Sorry, the edit suggestions are unrelated to your changes. It is not apparent, but I removed the ":" after "are". It would be great if you can remove the ":" after "are" from the other descriptions here as well.

Author: On it

- `stylistic()`, `historical-forms`, `styleset()`, `character-variant()`, `swash()`, `ornaments()`, `annotation()`
  - : Specifies the keywords and functions related to the [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures) longhand property.
- `small-caps`, `all-small-caps`, `petite-caps`, `all-petite-caps`, `unicase`, `titling-caps`
  - : Specifies the keywords and functions related to the [`font-variant-caps`](/en-US/docs/Web/CSS/font-variant-caps) longhand property.
- `<numeric-figure-values>`, `<numeric-spacing-values>`, `<numeric-fraction-values>`, `ordinal`, `slashed-zero`
  - : Specifies the keywords related to the [`font-variant-numeric`](/en-US/docs/Web/CSS/font-variant-numeric) longhand property. The possible values are: `lining-nums`, `oldstyle-nums`, `proportional-nums`, `tabular-nums`, `diagonal-fractions`, `stacked-fractions`, `ordinal`, and `slashed-zero`.
- `<east-asian-variant-values>`, `<east-asian-width-values>`, `ruby`

>   - : Specifies the keywords related to the [`font-variant-east-asian`](/en-US/docs/Web/CSS/font-variant-east-asian) longhand property. The possible values are: `jis78`, `jis83`, `jis90`, `jis04`, `simplified`, `traditional`, `full-width`, `proportional-width`, `ruby`.

Reviewer: ```suggestion
  - : Specifies the keywords related to the [`font-variant-east-asian`](/en-US/docs/Web/CSS/font-variant-east-asian) longhand property. The possible values are `jis78`, `jis83`, `jis90`, `jis04`, `simplified`, `traditional`, `full-width`, `proportional-width`, and `ruby`.
```

## See also

- [`text-transform`](/en-US/docs/Web/CSS/text-transform)
- [`text-combine-upright`](/en-US/docs/Web/CSS/text-combine-upright)
- [`text-orientation`](/en-US/docs/Web/CSS/text-orientation)
- [`font-variant-alternates`](/en-US/docs/Web/CSS/font-variant-alternates)
- [`font-variant-caps`](/en-US/docs/Web/CSS/font-variant-caps)
- [`font-variant-east-asian`](/en-US/docs/Web/CSS/font-variant-east-asian)
- [`font-variant-emoji`](/en-US/docs/Web/CSS/font-variant-emoji)
- [`font-variant-ligatures`](/en-US/docs/Web/CSS/font-variant-ligatures)
- [`font-variant-numeric`](/en-US/docs/Web/CSS/font-variant-numeric)

> - [`font-variant-position`](/en-US/docs/Web/CSS/font-variant-position)

Author: yeah that makes sense
